lib/ruby_terraform/commands/base.rb

```diff
@@ -37,22 +35,52 @@ def execute(opts = {})
       attr_reader :binary, :logger, :stdin, :stdout, :stderr
 
       def command_name
-        self.class.to_s.split("::")[-1].downcase
+        self.class.to_s.split('::')[-1].downcase
       end
 
-      def instantiate_builder
-        Lino::CommandLineBuilder
-            .for_command(binary)
-            .with_option_separator('=')
+      def do_before(_opts); end
```

Comment:
What does the `_` prefix mean here? I think I'd rather drop it if possible.

Reply:
Rubocop (or Rubymine - can't remember which) recommends it be used on any parameter that isn't actually used by the code in the method.